client/js/stores/month-store.js

```diff
@@ -1,9 +1,34 @@
-var Dispatcher = require('./dispatcher/dispatcher');
-var AppConstants = require('./constants/constants');
-var EventEmitter = require('event').EventEmitter;
+var Dispatcher = require('../dispatcher/dispatcher');
```

Comment:
This store doesn't appear to be registered with the Dispatcher. Just worth noting this isn't used yet!